src/BinaryBuilder.jl

```diff
@@ -4,15 +4,14 @@ using BinaryBuilderBase
 using ObjectFile
 using GitHub
 import InteractiveUtils
-using Pkg, Pkg.BinaryPlatforms, Pkg.PlatformEngines, Pkg.Artifacts
+using Pkg, Base.BinaryPlatforms, Pkg.Artifacts
```

Comment:
`Pkg.Artifacts` or `Artifacts`?

Reply:
We actually need to use `Pkg.Artifacts` because we do things like `bind_artifact!()` and such, which exist only in the `Pkg` version.